receiver/prometheusreceiver/testdata/config.yaml

```diff
@@ -4,6 +4,7 @@ prometheus/customname:
   buffer_count: 45
   use_start_time_metric: true
   start_time_metric_regex: '^(.+_)*process_start_time_seconds$'
+  report_extra_metrics: true
```

Comment:
I would suggest including "scrape" in the name, as it is a scrape.Option.  Possibly "report_extra_scrape_metrics".